corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/api.js

```diff
@@ -76,6 +76,28 @@ hqDefine("cloudcare/js/formplayer/menus/api", function () {
                                 return;
                             }
 
+                            Util.doUrlAction((urlObject) => {
+                                let update = false;
+                                // If we don't have an appId in the URL (usually due to form preview)
```

Comment:
form preview? should that be incomplete forms?

Reply:
...nevermind, I see you just moved this, though I still don't know what it means